backend/src/main/java/com/htecgroup/skynest/model/email/Email.java

```diff
@@ -1,18 +1,14 @@
 package com.htecgroup.skynest.model.email;
 
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @Getter
+@AllArgsConstructor
 public class Email {
   String to;
   String subject;
-  String emailBody;
+  String template;
+  String link;
```

Comment:
What happens if you need to sent a mail where no link option is needed? That option will be null ? Also what happens if there is email where you need to provide some user data or any other data that currently you don't support in this Email class?
Try to find more elegant and extendable solution for this kind of situations

Reply:
Yeah, this won't be too reusable in the future, as it is specific for the 2 emails we are currently sending. Will think of a better way.

Reply:
@petarstoimenovskihtec Would it be sufficient to have an array of String args, which could be filled with any info required by the email template you are using? So when you make a new email template which, for example, needs the users name, you can add it and use it in the template as args[2] ( or whichever index it is at ).

Reply:
@petarstoimenovskihtec Would it be sufficient to have an array of String args, which could be filled with any info required by the email template you are using? So when you make a new email template which, for example, needs the users name, you can add it and use it in the template as args[2] ( or whichever index it is at ).

Reply:
there is a saying in IT world.. when you are in doubt use Map :D